src/dispatch/static/dispatch/src/auth/Mfa.vue

```diff
@@ -0,0 +1,106 @@
+<template>
+  <v-container class="fill-height" fluid>
+    <v-row align="center" justify="center">
+      <v-col cols="12" sm="8" md="6" lg="4">
+        <v-card class="elevation-12">
+          <v-card-title class="text-center text-h5 py-4">
+            Multi-Factor Authentication
+          </v-card-title>
+          <v-card-text>
+            <v-alert v-if="status" :type="alertType" class="mb-4">
+              {{ statusMessage }}
+            </v-alert>
+            <v-btn
+              color="primary"
+              block
+              @click="verifyMfa"
+              :loading="loading"
+              :disabled="status === MfaChallengeStatus.APPROVED"
+            >
+              Verify MFA
+            </v-btn>
+          </v-card-text>
+        </v-card>
+      </v-col>
+    </v-row>
+  </v-container>
+</template>
+
+<script setup lang="ts">
+import { ref, computed } from "vue"
+import { useRoute } from "vue-router"
+import authApi from "@/auth/api"
+
+/* eslint-disable no-unused-vars */
```

Comment:
Was getting an `unused` here, but it definitely is used? Not sure what's up here. I added it to a type file and exported as const enum and got the same error.